langlib/lang.transaction/src/main/ballerina/transaction.bal

```diff
@@ -51,6 +51,20 @@ public type Timestamp readonly & object {
 
 # Returns information about the current transaction.
 #
+# ```ballerina
+# function createEntity() returns error? {
+#     transaction {
+#         transaction:Info info = transaction:info();
+#         check commit;
+#     }
+# }
+#
+# transactional function updateDB() returns error? {
```

Comment:
```suggestion
# transactional function updateDB() {
```
Shall we do this where possible error is not thrown from the body. Will need to send this tp 3.x as well